modules/base/src/properties/meshinformationproperty.cpp

```diff
@@ -43,7 +43,10 @@ MeshInformationProperty::MeshInformationProperty(std::string_view identifier,
                                                  std::string_view displayName,
                                                  InvalidationLevel invalidationLevel,
                                                  PropertySemantics semantics)
-    : CompositeProperty(identifier, displayName, invalidationLevel, semantics)
+    : CompositeProperty(
+          identifier, displayName,
+          "A CompositeProperty holding properties to show information about a mesh and its buffers"_help,
```

Comment:
I don't think the use needs to know that is is a CompositeProperty. 
"Information about a mesh and its buffers" would be better.